build warning on Mac

Common warning between Linux and Mac

```
In file included from yalantinglibs/include/ylt/metric.hpp:22:
yalantinglibs/include/ylt/metric/system_metric.hpp:185:41: warning: format specifies type 'long *' but the argument has type 'int64_t *' (aka 'long long *') [-Wformat]
  185 |   if (sscanf(result.c_str(), "%ld %ld", &resident, &virtual_size) != 2) {
      |                               ~~~       ^~~~~~~~~
      |                               %lld
yalantinglibs/include/ylt/metric/system_metric.hpp:185:52: warning: format specifies type 'long *' but the argument has type 'int64_t *' (aka 'long long *') [-Wformat]
  185 |   if (sscanf(result.c_str(), "%ld %ld", &resident, &virtual_size) != 2) {
      |                                   ~~~              ^~~~~~~~~~~~~
      |                                   %lld
```

--- a/include/ylt/metric/system_metric.hpp
+++ b/include/ylt/metric/system_metric.hpp
@@ -1,6 +1,7 @@
 #pragma once
 
 #include <chrono>
+#include <cinttypes>
 #include <cstdint>
 #include <cstdio>
 #include <fstream>
@@ -185,7 +186,8 @@ inline void stat_memory() {
     return;
   }
   const std::string& result = oss.str();
-  if (sscanf(result.c_str(), "%ld %ld", &resident, &virtual_size) != 2) {
+  if (sscanf(result.c_str(), "%" PRId64 " %" PRId64, &resident,
+             &virtual_size) != 2) {
     ELOG_WARN << "Fail to sscanf";
     return;
   }

